packages/mock-block-dock/src/use-mock-block-props.ts

```diff
@@ -158,23 +111,9 @@ export const useMockBlockProps = ({
     entities,
     links,
     linkedAggregationDefinitions,
-    startingEntity: latestBlockEntity,
+    startingEntity: blockEntity,
   });
 
-  // @todo we don't do anything with this type except check it exists - do we need to do this?
-  const latestBlockEntityType = useMemo(
-    () =>
-      entityTypes.find(
-        (entityType) =>
-          entityType.entityTypeId === latestBlockEntity.entityTypeId,
-      ),
-    [entityTypes, latestBlockEntity.entityTypeId],
-  );
-
```

Comment:
This is no longer needed. There are checks in use-mock-datastore that ensures the block always has an entityId and entityTypeId, even when they aren't present in the passed in blockEntity